tests/artist/read.js

```diff
@@ -0,0 +1,68 @@
+const { expect } = require("chai");
+const request = require("supertest");
+const getDb = require("../../src/services/db");
+const app = require("../../src/app");
+const testArtists = require("./testArtistData");
+
+describe("read artist", () => {
+  let db;
+  let dbArtists;
+
+  beforeEach(async () => {
+    db = await getDb();
+    await Promise.all(
+      testArtists.map(
+        async (testArtist) =>
+          await db.query("INSERT INTO Artist SET ?", testArtist)
+      )
+    );
+    [dbArtists] = await db.query("SELECT * from Artist");
+  });
+
+  afterEach(async () => {
+    await db.query("DELETE FROM Artist");
+    await db.end();
+  });
+
+  describe("/artist", () => {
+    describe("GET", () => {
+      it("returns all artist records in the database", async () => {
+        const res = await request(app).get("/artist").send();
+
+        expect(res.status).to.equal(200);
+
+        const responseArtists = res.body;
+
+        expect(responseArtists.length).to.equal(testArtists.length);
```

Comment:
Slight formatting tweak. I see what you were thinking by organising things in this way, but generally its better to have all your assertions together if at all possible. In this case you can gather your variables together then run your assertions in a block together as nothing will have been changed by this point
```
        const res = await request(app).get("/artist").send();
        const responseArtists = res.body;

        expect(res.status).to.equal(200);
        expect(responseArtists.length).to.equal(testArtists.length);
```